utils/cloud/aws/status_disk.yml

```diff
@@ -8,7 +8,7 @@
 
 - name: Disk status check
   template:
-    src: disk_status_check.j2
+    src: /utils/commondisk_status_check.j2
```

Comment:
Why are you putting this file in the parent folder utils instead of putting this inside common or other folders?

Reply:
done